posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -44,6 +44,8 @@ def calculate(self):
                 to_data("unique conversions", "unit", self._unsample(row[4]), self._unsample(row[5])),
                 to_data("conversion rate", "percentage", row[6], row[7]),
             ]
+            if self.query.includeRevenue:
+                results.append(to_data("conversion revenue", "currency", row[8], row[9]))
```

Comment:
This is a follow-up to my comment above, you're sometimes calculating revenue even if `includeRevenue` is false - when a conversion goal is set -, this feels like a bug then

Reply:
I don't think this is the case